docs/content/projects/oscal-deep-diff.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "OSCAL Deep Diff"
+description: "OSCAL Deep Diff - compare OSCAL artifacts programmatically and consistently"
```

Comment:
OSCAL Deep Diff - compares instead of compare (I am assuming is referring to the tool, not the actions a user may perform.

Reply:
Resolved in [f468d61](https://github.com/usnistgov/oscal-tools/pull/63/commits/f468d61bae90e0ca95dfc1685ee1feeb746bbddf)